auth-web/src/components/auth/TestimonialQuotes.vue

```diff
@@ -1,19 +1,47 @@
 <template>
   <section id="testimonials-container">
     <v-container class="quote-container">
-      <p>"We're incredibly excited to be able to incorporate as a Benefit Company in BC, it really reflects our values
-        and the role we want to play."</p>
-      <p><strong>&ndash; BC Benefit Company user</strong></p>
+      <div :class="{ 'fadeOut': isFading, 'fadeIn': !isFading}">
+        <p>{{quotes[quoteIndex].text}}</p>
+        <p><strong>&ndash; {{quotes[quoteIndex].author}}</strong></p>
+      </div>
     </v-container>
   </section>
 </template>
 
 <script lang="ts">
 import { Component, Emit, Prop, Vue } from 'vue-property-decorator'
-import LearnMoreButton from '@/components/auth/common/LearnMoreButton.vue'
 
 @Component({})
-export default class TestimonialQuotes extends Vue {}
+export default class TestimonialQuotes extends Vue {
+  private isFading: boolean = false
+  private quoteIndex: number = 0
+  private quotes: Array<any> =
+    [
+      {
+        text: 'We\'re incredibly excited to be able to incorporate as a Benefit Company in BC, it really reflects our values and the role we want to play.',
+        author: 'BC Benefit Company user'
+      },
+      {
+        text: 'This is a placeholder for a second quote.',
+        author: 'BC Benefit Company user'
+      },
+      {
+        text: 'This is a placeholder for third quote.',
+        author: 'BC Benefit Company user'
+      }
+    ]
+
+  mounted () {
+    setInterval(() => {
+      this.isFading = !this.isFading
+      setTimeout(() => {
+        this.isFading = !this.isFading
+        this.quoteIndex === this.quotes.length - 1 ? this.quoteIndex = 0 : this.quoteIndex++
```

Comment:
This confused me. Suggest you rewrite it as a normal if-then-else, or at least add parentheses around the comparison (which I initially thought was an assignment). 😛

Reply:
fair enough, ternary conditionals have a way of blending together. Will update this!

Reply:
Resolved.